tests/manage/pv_services/test_delete_project_with_pvc_and_pods.py

```diff
@@ -0,0 +1,165 @@
+        pvs = helpers.get_all_pvs()
+        space_used_before_deletion = check_ceph_used_space()
+        project_deletion_start_time = time.time()
+        #ocp.switch_to_default_rook_cluster_project() #TODO: Is this needed?
+        log.info("Deleting Project 1")
+        project_1.delete(resource_name=project_1.namespace)
+        log.info("Waiting for Project 1 deletion success...")
+        project_1.wait_for_delete(resource_name=project_1.namespace)
+        log.info("Project 1 deleted successfully.")
+        project_deletion_time = time.time() - project_deletion_start_time
+        log.info(
+            "Project 1 deletion time: {} seconds".format(project_deletion_time))
+        # PVs delete might still be running, check until all PVs are deleted
+        # Check time between first PV delete and last PV delete
+        time_between_pvcs_deletion_success = 0
+        # Also check delete time per each PV
+        individual_pv_delete_times = []
+        for pv in pvs:
+            pv_start_time = time.time()
+            pv_name = pv.backed_pv
+            #pv.delete()
+            pv.ocp.wait_for_delete(pv_name)
+            helpers.validate_pv_delete(pv_name)
+            pv_delete_time = time.time() - pv_start_time
+            individual_pv_delete_times.append({pv_name: pv_delete_time})
+            time_between_pvcs_deletion_success += pv_delete_time
+        log.info("{} seconds between first and last PVC deletion".format(
+            time_between_pvcs_deletion_success))
+        # Verify space has been reclaimed
+        space_used_after_deletion = check_ceph_used_space()
+        log.info("Verifying space has been reclaimed...")
+        assert space_used_after_deletion < space_used_before_deletion
+        log.info("Space reclaimed successfully.")
+
+        # Project 1 done, moving on to Project 2
+        log.info("Creating Project 2")
+        project_2 = project_factory()
+
+        log.info("Creating {} CephFS PVCs "
+                 "(each bound to an app pod)".format(self.cephfs_pvcs_num))
+        cephfs_pvcs_pods = \
+            [pod_factory(pvc=pvc_obj, interface=constants.CEPHFILESYSTEM)
+             for pvc_obj in multi_pvc_factory(interface=constants.CEPHFILESYSTEM,
+                                              project=project_2,
+                                              num_of_pvc=self.cephfs_pvcs_num)]
+        rbd_pvcs_pods = \
+            [pod_factory(pvc=pvc_obj, interface=constants.CEPHBLOCKPOOL)
+             for pvc_obj in multi_pvc_factory(interface=constants.CEPHBLOCKPOOL,
+                                              project=project_2,
+                                              num_of_pvc=self.rbd_pvcs_num)]
+        # Delete the entire Project 2 (along with all of its PVCs)
+        pvs = helpers.get_all_pvs()
+        space_used_before_deletion = check_ceph_used_space()
+        start_time = time.time()
+        # ocp.switch_to_default_rook_cluster_project() #TODO: Is this needed?
```

Comment:
yes, you will need to switch back to the default project

Reply:
No problem, will do.